nengo/spa/spa.py

```diff
@@ -88,7 +88,11 @@ def __setattr__(self, key, value):
 
     def get_module(self, name):
         """Return the module for the given name."""
-        return self._modules[name]
+        if name in self._modules:
+            return self._modules[name]
+        elif '_' in name:
+            module, name = name.rsplit('_', 1)
```

Comment:
And, just to clarify my ambiguous comment above:

> So this means that we can't have inputs and outputs names with an underscore in them, yes? 

Yes, but that's always been the case.  That's not something new to this PR.